### PR TITLE
Correctly parse uint64 vals in Contract Watcher

### PR DESCRIPTION
**Pull request by rmulhol**

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 11.5
```
Comment (elizabethengelman) [inline db/schema.sql]: just wanted to double check - do we want to commit this schema change in this PR?

File: pkg/contract_watcher/converter/converter_test.go [hunk 2]
```diff
@@ -26,19 +26,23 @@ import (
...
 
 var _ = Describe("Converter", func() {
-	var con *contract.Contract
-	var tusdWantedEvents = []string{"Transfer", "Mint"}
-	var ensWantedEvents = []string{"NewOwner"}
-	var marketPlaceWantedEvents = []string{"OrderCreated"}
-	var molochWantedEvents = []string{"SubmitVote"}
-	var err error
+	var (
+		fakeHeaderID = rand.Int63()
```
Comment (elizabethengelman) [inline pkg/contract_watcher/converter/converter_test.go]: 🎉

File: db/schema.sql (hunk 1 above)
Comment (rmulhol) [reply db/schema.sql]: I think so, but happy to take it out if folks disagree. My rationale was that our docker-compose script is using pg 12 and the schema in that repo is as well

**Comment by rmulhol** [comment]
> LGTM! :shipit:
> 
> Just curious - did you discover this issue while trying to test this out with the Oasis contract?

yep!

File: pkg/contract_watcher/converter/converter.go [hunk 3]
```diff
@@ -90,6 +90,9 @@ func (c *Converter) Convert(logs []gethTypes.Log, event types.Event, headerID in
 				if len(b) == 32 {
 					seenHashes = append(seenHashes, common.HexToHash(strValues[fieldName]))
 				}
+			case uint64:
```
Comment (Gslaughl) [inline pkg/contract_watcher/converter/converter.go]: Not really related to this PR, but the other thing I noticed about the relevant Oasis events is that they also take arguments of the `ERC20` type. I wonder how we'll want to handle those 🤔

Comment (rmulhol) [reply pkg/contract_watcher/converter/converter.go]: Based on a quick test locally, seems like it already parses `ERC20` vals correctly as addresses /shrug

**Pull request merged**